src/WinXP/index.js

```diff
@@ -334,13 +387,25 @@ function WinXP() {
     dispatch({ type: CANCEL_POWER_OFF });
   }
 
+  const resetScreenSaver = () => {
+    dispatch({
+      type: SCREEN_SAVER_PREVIEW,
+      payload: false,
+    });
+    setIsScreenSaverActive(false);
```

Comment:
This happens A LOT. I know moving it into the debounced `screenSaverIdleTime` causes problems, but still not fond of it 🤔